documentation/diff-reports-previous-version/AgentOntology.md

```diff
@@ -0,0 +1,188 @@
+
+- [Action Requirement](http://www.ontologyrepository.com/CommonCoreOntologies/ActionRequirement) [label](http://www.w3.org/2000/01/rdf-schema#label) "Process Requirement"@en 
+
+
+### Authority Role `http://www.ontologyrepository.com/CommonCoreOntologies/AuthorityRole`
+#### Removed
+- [Authority Role](http://www.ontologyrepository.com/CommonCoreOntologies/AuthorityRole) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A Role that is realized by Acts which create, modify, transfer, or eliminate Action Regulations or other Authority Roles, and inheres in an Agent in virtue of collective acceptance of that Agent's ability to issue binding directives."@en 
+
+#### Added
+- [Authority Role](http://www.ontologyrepository.com/CommonCoreOntologies/AuthorityRole) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A Role that is realized by Processes which create, modify, transfer, or eliminate Process Regulations or other Authority Roles, and inheres in an Agent in virtue of collective acceptance of that Agent's ability to issue binding directives."@en 
+
+
+### Domain of a Country `http://www.ontologyrepository.com/CommonCoreOntologies/Country`
+#### Removed
+- [Domain of a Country](http://www.ontologyrepository.com/CommonCoreOntologies/Country) [editorialNote](http://www.w3.org/2004/02/skos/core#editorialNote) "'Country' is an unclear term. Does it refer to the people, the government, the site, or the material in the site? Furthermore, what is properly referred to as a 'Country' is highly dependent on the classification practices of the international community. For example: The United Kingdom is a sovereign country that has Wales, a non-sovereign Country, as a member. Vatican City is a sovereign country that is landlocked by a city: Rome, Italy. Puerto Rico is not a county (it's a territory) but has a government, a distinct culture, and so on. Thus, here we do not definine 'Country' but instead define 'Domain of a Country'."@en 
+
+#### Added
+- [Domain of a Country](http://www.ontologyrepository.com/CommonCoreOntologies/Country) [editorialNote](http://www.w3.org/2004/02/skos/core#editorialNote) "'Country' is an unclear term. Does it refer to the people, the government, the site, or the material in the site? Furthermore, what is properly referred to as a 'Country' is highly dependent on the classification practices of the international community. For example: The United Kingdom is a sovereign country that has Wales, a non-sovereign Country, as a member. Vatican City is a sovereign country that is landlocked by a city: Rome, Italy. Puerto Rico is not a country (it's a territory) but has a government, a distinct culture, and so on. Thus, here we do not definine 'Country' but instead define 'Domain of a Country'."@en 
```

Comment:
The addition correctly replaces "county" with "country" but both include "definine" instead of "define" in the last sentence.